asv.conf.json

```diff
@@ -0,0 +1,39 @@
+{
+    "version": 1,
+    "project": "iris",
+    "project_url": "https://github.com/SciTools/iris",
+    "repo": ".",
+    "environment_type": "conda",
+    "show_commit_url": "http://github.com/scitools/iris/commit/",
+    //"pythons": ["2.7", "3.6", "3.7"],
```

Comment:
@trexfeathers This is `virtualenv` specific, right? If that's the case, how do we control the python versions per `conda` environment? 

Apologies, I'm an `asv` noob 😄

Reply:
@bjlittle ASV handles this with conda just fine - it just makes a different environment for each. I expect this to be something we might want to use but it seemed unnecessary for a minimal demonstration.

Reply:
@trexfeathers Okay, cool... but if we're not actively using the `pythons` key, then would you be adverse to removing it?

Reply:
Keeping commented code lines is another bad habit. I'll remove it.

Reply:
Between your expertise and the `asv` docs, I think we'll have it covered. Nice one, thanks!